### DISCUSSION TITLE
YC W20 company needs help! Deployments to Google Cloud fail with multiple instances.

### DISCUSSION THREAD

**Discussion by brennanerbz** [vercel/next.js#11093] [Help]
I'm deploying a next.js to Google Cloud app engine. Whenever I deploy the project to AE with multiple instances, it throws this error: net::err_aborted 404 for one of the core files.

Any advice? This is a huge problem.

**Comment by brennanerbz** [comment]
GET https://hyphy.tv/_next/static/gwNEOzop8Q8zCbcJBaQlm/pages/index.js net::ERR_ABORTED 404

**Comment by brennanerbz** [comment]
Happy to hop on a zoom with someone who can help. We're doing demo day tomorrow and need this fixed.

**Comment by Sabir001** [comment]
I am also facing same problem when there are 2 or more instances in my GCP instance group (Compute Engine). Did you find any solution yet? @brennanerbz

**Comment by MarceloAGuimaraes** [comment]
Go to the `next.config.js` and add (this will set you build id to be equal to the date in utc that you deployed the app):

```
generateBuildId: () => {
  const deployDate = new Date()
  return `${deployDate.toLocaleDateString('UTC').replaceAll('/', '-')}`
}
```
